cardano-db-sync/src/Cardano/DbSync/Default.hs

```diff
@@ -58,31 +56,23 @@ import           Ouroboros.Network.Block (blockNo)
 
 import           System.IO.Unsafe (unsafePerformIO)
 
--- | The default SyncNodePlugin.
--- Does exactly what the cardano-db-sync node did before the plugin system was added.
--- The non-default node takes this structure and extends the lists.
-defDbSyncNodePlugin :: SqlBackend -> SyncNodePlugin
-defDbSyncNodePlugin backend =
-  SyncNodePlugin
-    { plugOnStartup = []
-    , plugInsertBlock = [insertDefaultBlock backend]
-    , plugRollbackBlock = [rollbackToPoint backend]
-    }
-
--- -------------------------------------------------------------------------------------------------
-
 insertDefaultBlock
-    :: SqlBackend -> Trace IO Text -> SyncEnv -> [BlockDetails]
+    :: SyncEnv -> [BlockDetails]
     -> IO (Either SyncNodeError ())
-insertDefaultBlock backend tracer env blockDetails = do
+insertDefaultBlock env blockDetails = do
     thisIsAnUglyHack tracer (envLedger env)
```

Comment:
Can we re-visit the `thisIsAnUglyHack` function? 
The comments there suggest it only exists because of the split that is now going away. @erikd